news/2025/2025-06-05-osulazer-updates-june-5-2025.md

```diff
@@ -0,0 +1,74 @@
+- **A new mod has been to osu!taiko.** The Simplified Rhythm mod is exactly what it says on the tin, it turns complicated rhythms such as 1/8 and 1/6 patterns into 1/4, 1/3 rhythms into 1/2, and allows for further modifications via mod settings.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2025-06-05-simplified-rhythm.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Partial slider body transparency has been added to "argon" skins.** After some player complaints this has been manually added in by peppy, however in the future this will be configurable in-game on a skin-to-skin basis.
+
+![](/wiki/shared/news/2025-06-05-osulazer-updates-june-5-2025/argon-transparency.png)
+
+- **The "Require perfect hits" option has been added for the Mania Perfect mod.** For all the *true* perfectionists out there.
+
+### Editor
+
+- **The BSS process now has cool new sound effects.** After a complaint was made by a community member, [nekodex](https://osu.ppy.sh/users/102) cooked up brand new satisfying sounds that accompany you through the beatmap submission process.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2025-06-05-bss-sound-effect.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Distance snapping can no longer be used to place objects outside of playfield bounds.** The daredevils amongst the mapping community will need to find another way to no-clip out of bounds.
+- **Various editor crash fixes and improved logging and errors for BSS.** If you are a fellow nerd, you can check the [full changelog]() for the specific details.
+
+### Other
+
+- **A new "Tachyon" release stream has been added.** This is the equivalent of the "cutting edge" release stream from osu!(stable), but on osu!(lazer). With "Tachyon", you will be able to play on all pre-release versions of the game, and the devs are recommending you give it a try if you have been experiencing stutters on osu!(lazer).
+- **Plays with non-default mod configurations are now clearly displayed.** This means if a user set a score with Double Time, but edited the rate to something other than 1.5x, a new cog icon will appear on the top left of the mod icon. Hovering over the mod icon also gives more specific details.
+
+![](/wiki/shared/news/2025-06-05-osulazer-updates-june-5-2025/non-default-mod-config.png)
+
+- **You can now join multiplayer/playlist rooms from links.** If a link is shared on either in-game chat or anywhere outside the game, clicking on it now automatically makes you join.
+- **Replay playback issues have mostly been resolved.** If you are someone who's looking through replays on osu!(lazer), you should notice a huge different with this release.
```

Comment:
could maybe elaborate a bit on this? video mentions something about rewinding in mania no longer breaking combo.

if not, then apply this

```suggestion
- **Replay playback issues have mostly been resolved.** You'll notice that watching and seeking through replays should feel much better now.
```